### PR TITLE
Feat: Success story section for Flutter Page

### PR DESCRIPTION
**Pull request by cp-ashish-k**
## Review checklist

**Note:** Make sure all points should be checked before submiting PR for review, otherwise PR will be cosidered invalid.

- [x] Follow guidelines for creating branch name, adding commit messages or PR title
- [x] Add MR description if applicable
- [x] Follow guidelines for naming conventions everywhere (i.e files/folders, data structures)
- [x] Reuse code (if the same code is written twice, make it common and reuse it at both places)
- [x] Remove unused or commented code if not required
- [ ] Make MR self-approved - review MR as a reviewer and give it self-approval if everything is ok, if not make the required changes
- [ ] In progress MR should be marked as draft
- [ ] Make MR mark as ready before submitting it for review
- [x] Add images, videos, or gifs of UI
- [x] Add page speed score screenshot of UI (It should be > 85 for mobile and > 90 for desktops)

**Design should be tested on:**

- [x] Inspector (all iPhones and android devices like samsung, motorola)
- [x] Desktop (chrome, firefox, safari)
- [ ] Laptop / Macbook (chrome, firefox, safari)
- [x] iPhone(safari)
- [x] Available android devices (chrome, firefox)
- [x] iPad(safari)
- [x] Tablet (chrome, firefox)

- UI Image
   - Mobile UI   
image
   
  - Desktop UI  
image

- LightHouse Score for desktop
image
 
- Lighthouse score for mobile
image

### CODE REVIEW

**Comment by cp-dharti-r** [comment]
It should be sticky with the footer
Screenshot from 2023-11-09 17-09-50

File: nuxt-frontend/components/flutter-app-development/SuccessStorySection.vue [hunk 1]
```diff
@@ -0,0 +1,117 @@
+<template>
```
Comment (cp-dharti-r) [inline nuxt-frontend/components/flutter-app-development/SuccessStorySection.vue:1]: It should be sticky with the footer
Screenshot from 2023-11-09 17-09-50

File: nuxt-frontend/components/flutter-app-development/SuccessStorySection.vue [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+            :speed="3000"
+            :loop="true"
+            :spaceBetween="20"
+            :pagination="pagination"
+            :modules="modules"
+            class="swiper-container"
+          >
+            <swiper-slide v-for="(client, index) in clients" :key="index">
+              <div class="flex flex-col gap-6 lg:gap-10">
+                <div class="sub-h1-semibold">"{{ client.review }}"</div>
```
Comment (cp-jagruti-a) [inline nuxt-frontend/components/flutter-app-development/SuccessStorySection.vue:29]: all reviews starting line should be same its not like one start from above and one from below

**Comment by cp-dharti-r** [comment]
Also, let's start providing PR description and test steps (it will help the reviewer when PR is about fixing any issue)

File: nuxt-frontend/components/flutter-app-development/SuccessStorySection.vue [hunk 3]
```diff
@@ -0,0 +1,117 @@
+<template>
+  <div class="bg-black-core text-white text-center h-[550px] lg:h-[750px]">
```
Comment (cp-dharti-r) [inline nuxt-frontend/components/flutter-app-development/SuccessStorySection.vue:2]: Need to fix this as well
Screenshot from 2023-11-09 17-30-55

File: nuxt-frontend/components/flutter-app-development/SuccessStorySection.vue (hunk 2 above)
Comment (cp-ashish-k) [reply nuxt-frontend/components/flutter-app-development/SuccessStorySection.vue:29]: Unable to the point can you please elaborate more

Comment (cp-jagruti-a) [reply nuxt-frontend/components/flutter-app-development/SuccessStorySection.vue:29]: means top margin should be same for all reviews refer mobile app development what our client says section

File: nuxt-frontend/components/flutter-app-development/SuccessStorySection.vue [hunk 4]
```diff
@@ -0,0 +1,117 @@
+<template>
+  <div class="bg-black-core text-white text-center h-[600px] lg:h-[750px]">
+    <div
+      class="container flex gap-6 lg:gap-[4.5rem] flex-col pt-12 xs:pt-14 sm:pt-[4.5rem] lg:!pt-[7.5rem] xl:!pt-[6rem] lg:px-[12rem]"
```
Comment (cp-jagruti-a) [inline nuxt-frontend/components/flutter-app-development/SuccessStorySection.vue:4]: Use tailwind default

File: nuxt-frontend/components/flutter-app-development/SuccessStorySection.vue [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+    >
+      <div class="header-2">Our Success Stories</div>
+      <div class="flex flex-col gap-6 lg:gap-10">
+        <img
+          :src="quoteimage"
+          class="mx-auto w-[2.5rem] lg:w-[3.5rem] h-full object-contain"
```
Comment (cp-jagruti-a) [inline nuxt-frontend/components/flutter-app-development/SuccessStorySection.vue:10]: Use default classes here also

**Review by cp-jagruti-a — APPROVED** [review]

**Review by cp-jagruti-a — APPROVED** [review]

**Review by cp-dharti-r — APPROVED** [review]

**Pull request merged**